src/native/fs/fs_napi.cpp

```diff
@@ -213,6 +224,258 @@ struct Stat : public FSWorker
     }
 };
 
+/**
+ * Getxattr is an fs op
+ */
+struct Getxattr : public FSWorker
+{
+    std::string _path;
+    std::vector<KeyValue> _md;
+    Getxattr(const Napi::CallbackInfo& info) 
+        : FSWorker(info)
+        , _md(0)
+    {
+        _path = info[1].As<Napi::String>();
```

Comment:
I would send the prefix as arg (optional) here, so this will filter only the xattrs matching the prefix.